### ISSUE TITLE
When building for Android/Quest - User is prompted to perform an "Entitlement Check" - this appears to be a Pico dependency

### ISSUE THREAD

**Issue by SimonDarksideJ** [realitycollective/realitytoolkit.dev#4]
# Reality Collective - Reality Toolkit Bug Report

## Describe the bug

When building for Android/Quest - User is prompted to perform an "Entitlement Check" - this appears to be a Pico dependency

## To Reproduce

1. Clone RCT.Dev
2. Switch to Android
3. Build

## Expected behavior

PICO should probably only be activated for Pico builds

## Actual behavior

Pico Dependencies are firing always, is this intended?

**Comment by FejZa** [comment]
This is not a bug. It's a feature of the Pico SDK and if someone pulls test SDK in then they should be capable of answering test Dialog.

**Comment by SimonDarksideJ** [comment]
True, but it is firing all the time.  Even when building for Quest.

SO if it is a generic feature, should we import and override it>?

**Comment by FejZa** [comment]
Yeah, it pops up even when building for UWP. But there is a "Don't ask again" option, so it's really a matter of answering it once and then not dealing with it again which I see in the responsibility of the developer and not us. Even in my own projects I had to answer this dialog and be done with it. Not a big deal. I don't see this as an issue of the toolkit.

**Comment by SimonDarksideJ** [comment]
Fair enough, but I'm not a big fan of "just introducing" something that most might object to by just having the toolkit and a platform installed, better we try and introduce it ourselves as a feature and override the setting they are checking for.
